news/2023/2023-07-06-introducing-new-beatmap-pack-categories.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
+date: 2023-07-06 09:00:00 +0000
+---
+
+Check out the latest and greatest new beatmap pack categories!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/banner.png)
+
+I like maps. You like maps. We all like maps.
```

Comment:
Sentences that are in first person like this are generally avoided, however I would like to hear more thoughts about this. I know that I generally avoid saying stuff in the first person in a news post as it usually got changed before publishing

Reply:
would this work better just changing the whole line just to 'Everyone wants more maps.'?